### PR TITLE
Proofread the docs

### PR DESCRIPTION
**Pull request by camuthig**

### CODE REVIEW

File: docs/event_store.md [hunk 1]
```diff
@@ -32,20 +32,20 @@ event store methods. The following events are available (event target is always
 - `fetchStreamMetadata`: event params: `streamName` - result params: `metadata`, `streamNotFound`
 - `updateStreamMetadata`: event params: `streamName`, `metadata` - result params: `streamNotFound`
 - `fetchStreamNames`: event params: `filter`, `metadataMatcher`, `limit`, `offset` - result params: `streamNames`
-- `fetchStreamNamesRegex`: event params: `filter`, `metadataMatcher`, `limit`, `offset` - result params: `streamNames` 
+- `fetchStreamNamesRegex`: event params: `filter`, `metadataMatcher`, `limit`, `offset` - result params: `streamNames`
 - `fetchCategoryNames`: event params: `filter`, `offset`, `limit` - result params: `categoryNames`
 - `fetchCategoryNamesRegex`: event params: `filter`, `offset`, `limit` - result params: `categoryNames`
 
-If the event store implements additionally \Prooph\EventStore\CanControlTransactionActionEventEmitterAwareEventStore,
+If the event store implements \Prooph\EventStore\CanControlTransactionActionEventEmitterAwareEventStore,
```
Comment (prolic) [inline docs/event_store.md]: That class name is wrong (outdated)

File: docs/migration.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
 
 ## Moved classes
 
-The `Prooph\Snapshot\*` classes are now moved to its own repository [SnapshotStore](https://github.com/prooph/snapshot-store).
+The `Prooph\Snapshot\*` classes are now moved to their own repository, [SnapshotStore](https://github.com/prooph/snapshot-store).
```
Comment (prolic) [inline docs/migration.md]: Prooph\EventStore\Snapshot probably?

**Comment by camuthig** [comment]
@prolic let me know if you need me to squash and force push to get it to one commit.

**Comment by prolic** [comment]
That's fine, thanks a lot.

**Comment by codeliner** [comment]
@camuthig thank you very much for proofread the docs. You know that I'll ping you now whenever new docs are written ;) Here is a buch of new stuff that would benefit from a proofread as well: https://github.com/prooph/proophessor/tree/master/docs/tutorial

Just in case you're looking for more :D (would be really awesome)

**Pull request merged**